ydb/docs/en/core/dev/secondary-indexes.md

```diff
@@ -82,6 +88,12 @@ FROM series VIEW views_index
 WHERE views = 0;
 ```
 
+{% note info %}
+
+Currently, deleting data is possible only using a synchronous secondary index. This is because data removal is possible only in [Serializable](../concepts/transactions.md#modes) transactions, but accessing asynchronous indices would break the guarantees of this transaction mode.
```

Comment:
```suggestion
Currently, deleting data is possible only using a synchronous secondary index. This is because data removal is permitted only in [Serializable](../concepts/transactions.md#modes) transactions, and accessing asynchronous indices would violate the guarantees of this transaction mode.
```